tests/aws/services/stepfunctions/v2/services/test_aws_sdk_task_service.py

```diff
@@ -139,6 +139,63 @@ def test_dynamodb_put_update_get_item(
             exec_input,
         )
 
+    @markers.snapshot.skip_snapshot_verify(
+        paths=[
+            # TODO: aws-sdk SFN integration now appears to be inserting decorated error names into the cause messages.
+            #  Upcoming work should collect more failure snapshot involving other aws-sdk integrations and trace a
+            #  picture of generalisability of this behaviour.
+            #  Hence insert this into the logic of the aws-sdk integration.
+            "$..cause"
+        ]
+    )
+    @markers.aws.validated
+    def test_sfn_send_task_success_no_such_token(
```

Comment:
success and failure could be two parameters of a parametrized test with identical code. See parametrization comment in https://github.com/localstack/localstack/pull/10168